src/Shared/HandyControlDemo_Shared/Service/Data/DataService.cs

```diff
@@ -86,7 +88,11 @@ internal List<AvatarModel> GetContributorDataList()
             try
             {
                 var json = client.DownloadString(new Uri("https://api.github.com/repos/nabian/handycontrol/contributors"));
+#if !Core
                 var objList = JsonConvert.DeserializeObject<List<dynamic>>(json);
+#else
+                var objList = System.Text.Json.JsonSerializer.Deserialize<List<dynamic>>(json);
```

Comment:
core 3.0 does not support for serializing dynamic objects for the time being.

Reply:
Update to net core 3 preview 7 released  yesterday

Reply:
yeah, but with preview 7, an exception will be thrown here.